voluncheer/opportunityboard/tests_views.py

```diff
@@ -1,10 +1,69 @@
+import datetime
+
 from django.test import TestCase
 from django.urls import reverse
 
+from opportunityboard.models import Category
+from opportunityboard.models import Opportunity
+from opportunityboard.models import Subcategory
+from opportunityboard.models import Subsubcategory
+from profiles.models import Organization
+from profiles.models import User
+from profiles.models import UserType
+
 
 class OpportunityboardTestCase(TestCase):
     """Test cases for Opportunityboard view"""
 
+    @classmethod
```

Comment:
Why is this looking just alike test_model above? Isn't it redundant?

Reply:
Because the test data from test_models doesn't carry over to test_views, we need to recreate it. In general, we should expect some repetition in unit tests.

Reply:
Since the test db is made from scratch, I needed at least one opportunity in order to test the `update_an_opportunity` url, which is why I simply copied the code over. I'll include a comment to clarify